pom.xml

```diff
@@ -105,6 +103,11 @@
       <version>1.18.16</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.9.0</version>
+    </dependency>
```

Comment:
can you undo changes to this file? (also, Gson is optional and for consumer if they want to use it)